examples/gallery/embellishments/inset_rectangle_region.py

```diff
@@ -0,0 +1,48 @@
+"""
+Inset map showing a rectangle region
+------------------------------------
```

Comment:
```suggestion
Inset map showing a rectangular region
--------------------------------------
```
Actually wondering whether to use 'box' region instead :thinking:

Reply:
😀rectangle is more accurate for this example, right?  we actually plot a so-called rectangle using `+r` argument.

Reply:
Well, if we define a rectangle as having 90 degree angles :wink: But let's not do too much bikeshedding on this topic.